cli/mobycli/scout_suggest.go

```diff
@@ -53,10 +55,14 @@ func displayScoutQuickViewSuggestMsg(image string) {
 	}
 	out := os.Stderr
 	b := color.New(color.Bold)
-	_, _ = fmt.Fprintln(out)
-	_, _ = b.Fprintln(out, "What's Next?")
-	_, _ = fmt.Fprintf(out, "  View summary of image vulnerabilities and recommendations → %s", color.CyanString("docker scout quickview%s", image))
-	_, _ = fmt.Fprintln(out)
+
+	_, _ = b.Fprintln(out, "\nWhat's Next?")
```

Comment:
it was already there! :p

Reply:
Sorry, that's me 😇 
(I often prefer that to a `//nolint:errcheck` but maybe there's a better/cleaner way)

Reply:
I didn't even pay it any mind at the time, but now that I think about it it's probably a smell - we should be checking if it errors, since if an earlier message fails to write we probably don't want to send the subsequent ones.

Reply:
Yeah, these are annoying; you can configure the linter to ignore these specific ones in the config, or add `//nolint` tags, but that's noisy as well.

Using `_` is the most "correct" option, and it _does_ indicate _"no, this was not a mistake: I heard you complain about error-handling, and I decided to ignore you 😂"_

That said, it's really cumbersome; especially because a lot of printing is happening when you work on a CLI; I was exploring some alternatives in https://github.com/docker/cli/pull/3797 (think I have an  "approach 2" branch somewhere), but haven't found a "great" way yet.